src/main/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtil.java

```diff
@@ -486,6 +490,21 @@ public static GetRefreshUsersResponse buildEmptyGetRefreshUsersResponse() {
         return response;
     }
 
+    public static long generateRecordNumberAsNanoSec(LocalDateTime dateTime) {

```

Comment:
have you added tests for these?

Reply:
have added UT for `generateRecordNumberAsNanoSec` & `convertRecordNumberToLocalDateTime` in commit [74f6a85](https://github.com/hmcts/rd-professional-api/pull/1529/commits/74f6a85f462bd2c50e8959bd0873a8d031fbd712)